### ISSUE TITLE
Let CA1068 not report on a method taking IProgress as the last parameter

### ISSUE THREAD

**Issue by Tan90909090** [dotnet/roslyn-analyzers#2281]
#### Analyzer package

[Microsoft.CodeAnalysis.FxCopAnalyzers](https://www.nuget.org/packages/Microsoft.CodeAnalysis.FxCopAnalyzers)

#### Package Version

v2.9.1

#### Diagnostic ID

CA1068

#### Repro steps
```cs
using System;
using System.Threading;
using System.Threading.Tasks;

public class C
{
    public Task SomeAsync(CancellationToken cancellationToken, IProgress<int> progress)
    {
        throw new NotImplementedException();
    }
}
```

#### Expected behavior
CA1068 is not reported.
Because some Microsoft's documentations take IProgress as the last parameter:
1. https://docs.microsoft.com/en-us/dotnet/standard/asynchronous-programming-patterns/implementing-the-task-based-asynchronous-pattern#io-bound-tasks
1. https://docs.microsoft.com/en-us/dotnet/standard/asynchronous-programming-patterns/task-based-asynchronous-pattern-tap#choosing-the-overloads-to-provide

#### Actual behavior
CA1068 is reported:
`Method 'C.SomeAsync(System.Threading.CancellationToken, System.IProgress<int>)' should take CancellationToken as the last parameter`
